wallet/src/Pos/Wallet/Web/Methods/History.hs

```diff
@@ -39,24 +39,15 @@ import           Pos.Wallet.Web.State       (AddressLookupMode (Ever), addOnlyNe
                                              getHistoryCache, getPendingTx, getTxMeta,
                                              getWalletPendingTxs, setWalletTxMeta)
 import           Pos.Wallet.Web.Util        (decodeCTypeOrFail, getAccountAddrsOrThrow,
-                                             getWalletAccountIds, getWalletAddrsSet,
-                                             getWalletAddrs)
+                                             getWalletAccountIds, getWalletAddrs,
+                                             getWalletAddrsSet)
 
 
 getFullWalletHistory :: MonadWalletWebMode m => CId Wal -> m (Map TxId (CTx, POSIXTime), Word)
 getFullWalletHistory cWalId = do
     addrs <- mapM decodeCTypeOrFail =<< getWalletAddrs Ever cWalId
-
     unfilteredLocalHistory <- getLocalHistory addrs
-
-    blockHistory <- getHistoryCache cWalId >>= \case
-        Just hist -> pure hist
-        Nothing -> do
-            logWarning $
```

Comment:
Should we worry if it's empty? 🤔

Reply:
No, it's common case for newly created wallets. Don't understand your concern

Reply:
I mean: was this `logWarning` here by mistake or you just deleted it for another reason? So apparently it was here by mistake, ok.

Reply:
Ah, yep